docs/release/release-tasks.md

```diff
@@ -292,40 +301,40 @@ The goal of this task to make the book for the current release available under e
 3. Update references in introduction.md only on the main branch (drop unsupported versions, add the new release version).
    <br>Prior art: [Add release 1.2 book link](https://github.com/kubernetes-sigs/cluster-api/pull/6697)
 
-#### Polish release notes
+#### Create PR for release notes
 
-1. Checkout the latest commit on the release branch, e.g. `release-1.4`.
+1. Checkout the latest commit on the main branch.
```

Comment:
this is important: the notes _must_ be generated from and checked into the main branch

Reply:
That's a good question. @nawazkh @furkatgofurov7 does `notes.go` always need to be run from the commit we're releasing from?

Reply:
Ok, I checked and you were correct @sbueringer, the notes tool will generate notes from `PREVIOUS_TAG` to `HEAD` so this means base notes need to be generated from the release branch still, clarified the instructions.

The PR should still be open in the main branch.

```
		commitRange = lastTag()
		cmd = exec.Command("git", "rev-list", commitRange+"..HEAD", "--merges", "--pretty=format:%B") //nolint:gosec
```